compiler/rustc_mir_transform/src/coroutine.rs

```diff
@@ -725,6 +727,10 @@ struct LivenessInfo {
     /// Which locals are live across any suspension point.
     saved_locals: CoroutineSavedLocals,
 
+    /// Which locals are live *and* initialized across any suspension point.
+    /// A local that is live but is not initialized does not need to accounted in auto trait checking.
```

Comment:
Your friendly neighborhood spell checker 😁 
```suggestion
    /// A local that is live but is not initialized does not need to be accounted for in auto trait checking.
```